packages/html/src/SimpleTable.ts

```diff
@@ -58,7 +69,10 @@ export class SimpleTable extends HTMLWidget {
 SimpleTable.prototype._class += " html_SimpleTable";
 
 export interface SimpleTable {
+    renderHtmlDataCells(): boolean;
+    renderHtmlDataCells(_: boolean): this;
```

Comment:
Does this match the equivalent name in Table and dgrid.Table?

Reply:
It matches `other_Table`, but I don't think `dgrid_Table` has that feature yet.

Reply:
Thats fine.